pkg/commands/aclentry/aclentry_test.go

```diff
@@ -51,7 +51,23 @@ func TestACLEntryCreate(t *testing.T) {
 				},
 			},
 			Args:       args("acl-entry create --acl-id 123 --ip 127.0.0.1 --service-id 123"),
-			WantOutput: "Created ACL entry '456' (ip: 127.0.0.1, service: 123)",
+			WantOutput: "Created ACL entry '456' (ip: 127.0.0.1, negated: false, service: 123)",
+		},
+		{
```

Comment:
@doramatadora new test to validate the fix.

Reply:
otherwise I think all other changes are the generic change of path from `/v4` to `/v5`.